lib/dump.txt

```diff
@@ -0,0 +1,58 @@
+#100000000.times do
```

Comment:
It's good practice not to include commented out/unused code in a PR to main. We only want production ready code merged in.